tests/UI/campaigns/functional/FO/03_userAccount/03_getGdprDataOnPdf.js

```diff
@@ -0,0 +1,96 @@
+require('module-alias/register');
+
+const {expect} = require('chai');
+
+const helper = require('@utils/helpers');
+const files = require('@utils/files');
+
+// Import data
+const {DefaultAccount} = require('@data/demo/customer');
+
+// Importing pages
+const foHomePage = require('@pages/FO/home');
+const foLoginPage = require('@pages/FO/login');
+const foMyAccountPage = require('@pages/FO/myAccount');
+const foPersonalGdprDataPage = require('@pages/FO/myAccount/personalGdprData');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_FO_userAccount_getGdprDataOnPdf';
+
+// Init const
+let pdfFilePath;
+
+let browserContext;
+let page;
+
+/*
+Sign in with default account
```

Comment:
You should check that the module GPDR is installed as it is not a project module